packages/AutoTDD-GUI.package/ATDDBrowser.class/class/globalFunctionalityDictionary.st

```diff
@@ -0,0 +1,11 @@
+constants
+globalFunctionalityDictionary
+	
+	^ OrderedDictionary
+		newFrom: {
+			'Pause All' -> #pauseAll.
+			'Resume All' -> #resumeAll.
+			'Run All' -> #runAll.
+			'Create Overview' -> #openOverview.
+      		'toggle_sound_placeholder' -> #toggleSounds.
```

Comment:
This should probably be something like `Toggle Sound`. Even better would be a solution that shows `Toggle Sound (On/Off)` depending on the current state.

Check out `soundsButtonLabel` and `toggleSounds`; might help

Reply:
Addressed in #67